schema.js

```diff
@@ -0,0 +1,119 @@
+{
+    "_id": "getty_WA0682-001",
```

Comment:
The "_id" concept is another open question. I'll add that to the questions list at the top.